### PR TITLE
11.0 fix seq chm

### PR DESCRIPTION
**Pull request by Gorash**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by wtaferner** [comment]
Hi Daddy ;-) Great to be back on the topic...all the Ninja turtles are hopefully eating their Pizza soon in the correct order even if the do some acrobatic moves during lunch time ;-)

**Comment by Gorash** [comment]
@ged-odoo ping review

File: addons/web/static/src/js/views/list/list_editable_renderer.js [hunk 1]
```diff
@@ -547,17 +547,47 @@ ListRenderer.include({
...
+        var reorderAll = false;
+        var sequence = (asc ? -1 : 1) * Infinity;
+        _.each(rows, function (row, index) {
+            if (index < lower && ((asc && sequence >= row.data[self.handleField]) || (!asc && sequence <= row.data[self.handleField]))) {
+                reorderAll = true;
+            }
+            if(index === lower) {
+                sequence = _.min(_.pluck(_.pluck(rows.slice(lower, number), 'data'), self.handleField)) + number;
+            }
+            if(index >= upper && ((asc && sequence >= row.data[self.handleField]) || (!asc && sequence <= row.data[self.handleField]))) {
```
Comment (adr-odoo) [inline addons/web/static/src/js/views/list/list_editable_renderer.js]: Same condition as before? As discussed, I think it shouldn't work. + add a test

File: addons/web/static/src/js/views/basic/basic_model.js [hunk 2]
```diff
@@ -3683,6 +3692,20 @@ var BasicModel = AbstractModel.extend({
             return list;
         });
     },
+    _selectDataList: function (list) {
```
Comment (ged-odoo) [inline addons/web/static/src/js/views/basic/basic_model.js]: docstring

**Comment by hugho-ad** [comment]
@Gorash  Can you share us the fix commit?

**Comment by Gorash** [comment]
https://github.com/odoo/odoo/commit/b7ee1aa4b1ea906195c234020cb708a1da9de6a7

**Pull request closed without merging**